src/xcode/ENA/ENA/Source/Services/CoronaTestService/__tests__/CoronaServiceTests.swift

```diff
@@ -0,0 +1,408 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+@testable import ENA
+import ExposureNotification
+import XCTest
+
+class CoronaServiceTests: XCTestCase {
```

Comment:
I renamed this file to `CoronaTestServiceTests`, I think this might be a duplicate from the merge and can probably just be removed

Reply:
So what should I do here?

Reply:
Yes, seems like it is not even in the project, just in the file system. Just delete it 👍